File: 03.Functions-And-Statements-Exercise/oddEvenSum.js
function oddEvenSum(number) {
    let numberAsString = String(number)
    let oddSum = 0
    let evenSum = 0

    for (const num of numberAsString) {
        if (num % 2 === 0) {
            evenSum += Number(num)
        } else {
            oddSum += Number(num)
        }
    }

    console.log(`Odd sum = ${oddSum}, Even sum = ${evenSum}`)
}

oddEvenSum(1000435)
oddEvenSum(3495892137259234)

File: 03.Functions-And-Statements-Exercise/odd_even_sum.py
def odd_even_sum(number):
    number_as_string = str(number)
    odd_sum = 0
    even_sum = 0

    for num in number_as_string:
        if int(num) % 2 == 0:
            even_sum += int(num)
        else:
            odd_sum += int(num)

    print(f"Odd sum = {odd_sum}, Even sum = {even_sum}")


odd_even_sum(1000435)
odd_even_sum(3495892137259234)
